molecule/docker-base/verify.yml

```diff
@@ -0,0 +1,32 @@
+- name: Verify
+  hosts: all
+  gather_facts: false
+  tasks:
+    - name: Populate service facts
+      ansible.builtin.service_facts:
+
+    - name: Check that expected services are up and running
+      ansible.builtin.assert:
+        quiet: true
+        that:
+          - ansible_facts.services[item] is defined
+          - ansible_facts.services[item].state == 'running'
+      loop:
+        - activemq.service
```

Comment:
missing postgresql

Reply:
damn you noticed 😀 pg service name depends on OS - didn't want to bother with that here - it's worth it?

Reply:
not really. No db will anyway prevent most other services from reporting a running status